doc/iterations/2025.1/add_support_for_model_publication.adoc

```diff
@@ -0,0 +1,80 @@
+= Add support for model publication
+
+== Problem
```

Comment:
Based on the previous comments, I assume this would be an implementation of a workflow tailored to Studios. To me this whole feature looks like a different shape if we go for a solution where Sirius Web only provides the workflow execution mechanism.